### PR TITLE
fixed initials update and documentation

### PR DESCRIPTION
**Pull request by acaldas**
https://github.com/ripe-tech/ripe-core/issues/3092

screen shot 2018-03-19 at 17 22 08

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,7 +134,28 @@ ripe.getPrice(function(value) {
...
+```javascript
+ripe.setInitials("SW", "metal_gold");
+
+ripe.bindImage(document.getElementById("frame-initials"), {
+    showInitials: true
+});
+```
+
+ripe.setInitials(function(value) {
+    var price = document.getElementById("price");
```
Comment (hugo-gomes) [inline README.md]: review this e

File: src/python/ripe_demo/templates/partials/layout.demo.html.tpl [hunk 2]
```diff
@@ -22,5 +22,8 @@
             <img id="frame-6" data-size="200" src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" />
             <img id="frame-top" data-size="200" src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" />
         </div>
+        <div class="frame-initials-container">
```
Comment (joamag) [inline src/python/ripe_demo/templates/partials/layout.demo.html.tpl]: name it just initials-container

File: src/python/ripe_demo/templates/partials/layout.demo.html.tpl [hunk 3]
```diff
@@ -22,5 +22,8 @@
             <img id="frame-6" data-size="200" src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" />
             <img id="frame-top" data-size="200" src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" />
         </div>
+        <div class="frame-initials-container">
+            <img id="frame-initials" class="frame-initials" width="250" height="250" />
```
Comment (joamag) [inline src/python/ripe_demo/templates/partials/layout.demo.html.tpl]: just initials for both id and class

File: src/python/ripe_demo/static/js/main.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ window.onload = function() {
     var element = document.getElementById("configurator");
     var _body = document.querySelector("body");
     var url = _body.dataset.url || "https://sandbox.platforme.com/api/";
-    var brand = _body.dataset.brand || "myswear";
+    var brand = _body.dataset.brand || "swear";
```
Comment (joamag) [inline src/python/ripe_demo/static/js/main.js]: :+1:

File: src/python/ripe_demo/static/js/main.js [hunk 5]
```diff
@@ -68,6 +68,13 @@ window.onload = function() {
         image.setFrame("9");
     });
 
+    ripe.setInitials("SW", "metal_gold");
+    var initialsImage = document.getElementById("frame-initials");
+    initialsImage.classList.add(model);
```
Comment (joamag) [inline src/python/ripe_demo/static/js/main.js]: why do we need to add an extra class ?

File: README.md [hunk 6]
```diff
@@ -134,7 +134,22 @@ ripe.getPrice(function(value) {
...
+If your initials require a transformation to different profiles you can set a function that receives the `initials` and `engraving` parameters and transforms it into a map with initials and an array of profiles using the `setInitialsBuilder` function.
+
+```javascript
+ripe.setInitials("SW", "metal_gold");
+
+ripe.bindImage(document.getElementById("frame-initials"), {
+    showInitials: true
+});
+```
+
```
Comment (joamag) [inline README.md]: remove extra line

**Pull request merged**